packages/react-mutation-mapper/src/store/DefaultMutationMapperDataFetcher.ts

```diff
@@ -34,7 +38,11 @@ import {
 } from '../util/DataFetcherUtils';
 import { getMyVariantInfoAnnotationsFromIndexedVariantAnnotations } from '../util/VariantAnnotationUtils';
 import { uniqueGenomicLocations } from '../util/MutationUtils';
-import { getEvidenceQuery } from '../util/OncoKbUtils';
+import { OncoKbAnnotationQuery } from '../../../../src/shared/lib/StoreUtils';
```

Comment:
We cannot import dependencies from `cbioportal-frontend` workspace, we need to move `OncoKbAnnotationQuery` into `cbioportal-frontend-commons` first